src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineCaseFieldDescriptor.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor;
+
+use JMS\Serializer\Annotation\ExclusionPolicy;
+
+/**
+ * This class defines the necessary information for a field to resolve it within a Doctrine Query for the ListBuilder.
+ *
+ * @ExclusionPolicy("all")
+ */
+class DoctrineCaseFieldDescriptor extends AbstractDoctrineFieldDescriptor
+{
+    /**
+     * @var DoctrineCaseDescriptor
+     */
+    private $case1;
+    /**
+     * @var DoctrineCaseDescriptor
+     */
+    private $case2;
+
+    public function __construct(
+        $name,
+        DoctrineCaseDescriptor $case1,
+        DoctrineCaseDescriptor $case2,
```

Comment:
What do you mean by not really? At least I would introduce a `DoctrineDescriptor` instead a `DoctrineCaseDescriptor`. The `DoctrineDescriptor` these three properties (which are the most basic ones), and the `DoctrineFieldDescriptor` inherits from the `DoctrineDescriptor`.